README.md

```diff
@@ -64,24 +64,37 @@ inside the `api-specification` package.
 
 ### Release
 
-To release the packages to npm, run
+To prepare for a release, you will need a fresh clone of the `master` branch of `symphonyoss/ContainerJS`.
```

Comment:
Does "fresh clone" mean you have to start in a blank folder and clone the repo from scratch? Or is it sufficient to just make sure your `master` branch is up to date with the remote?